src/chain.h

```diff
@@ -228,6 +230,13 @@ class CBlockIndex
         return proof.value();
     }
 
+    const bool dynafed_block() const {
```

Comment:
Clang says

    'const' type qualifier on return type has no effect

Reply:
yeah this warning has been annoying me, I thought it was fixed on the 22.x branch but it is not

Reply:
Thank you for posting this, but it has no effect on my RPI4 even though I have the latest version. On startup I get the same assert error. Any tips for me?

Reply:
If you see the assertion on startup, it usually means that you started IBD without `trim_headers` and then enabled `trim_headers` without finishing IBD, so the headers chain is much longer than the blocks. Either way, let's continue that conversation in https://github.com/ElementsProject/elements/issues/1240